julia/LibCEED.jl/docs/make.jl

```diff
@@ -0,0 +1,28 @@
+using Documenter, LibCEED, LinearAlgebra
+
+makedocs(sitename="LibCEED.jl Docs",
+         format=Documenter.HTML(prettyurls=false),
+         pages=[
+             "Home" => "index.md",
+             "Ceed Objects" => [
+                "Ceed.md",
+                "CeedVector.md",
+                "ElemRestriction.md",
+                "Basis.md",
+                "QFunction.md",
+                "Operator.md",
+             ],
+             "Utilities" => [
+                "Misc.md",
+                "Globals.md",
+                "Quadrature.md",
+             ],
+             "C.md",
+             "UserQFunctions.md",
+             "Examples.md",
+         ])
+
+deploydocs(
+    repo="github.com/CEED/libCEED-julia-docs.git",
+    devbranch="will/julia",
```

Comment:
The `devbranch` here will need to be changed to main when we merge.

Reply:
Done in the merge.